server/lib/queries.js

```diff
@@ -36,7 +36,7 @@ const getTotalDonations = () => {
     SELECT SUM(${generateFXConversionSQL()}) AS "totalDonationsInUSD"
     FROM "Transactions"
     WHERE amount > 0 AND "PaymentMethodId" IS NOT NULL
-  `.replace(/\n/g, ' '), // this is to remove the new lines and save log space.
+  `.replace(/\s\s+/g, ' '), // this is to remove the new lines and save log space.
```

Comment:
It will. That won't kill the CPU ;-)
But your solution will swap 2 or 3 spaces with 2 spaces instead of one.

No big deal either way ;-)